tendrl/node_agent/manager/manager.py

```diff
@@ -239,6 +243,35 @@ def on_pull(self, raw_data):
                     self.etcd_client.write(("nodes/%s/Disks/free/%s") % (
                         raw_data["node_id"], disk), "")
 
+    def execute_discovery_plugins(self, node_id):
```

Comment:
Can be renamed as `load_and_execute_platform_discovery_plugins` and no need to pass `node_id` as parameter. We can always say `utils.get_local_node_context()`